packages/amplify_api/ios/Classes/ApiBridge.swift

```diff
@@ -35,18 +35,28 @@ class ApiBridge {
         return Amplify.API.delete(request: request, listener: listener)
     }
     
-    
-    func query<R: Decodable>(request: GraphQLRequest<R>,
+    func head(request: RESTRequest, listener: ((AmplifyOperation<RESTOperationRequest, Data, APIError>.OperationResult) -> Void)?) -> RESTOperation?{
```

Comment:
Can we use a typealias for: `((AmplifyOperation<RESTOperationRequest, Data, APIError>.OperationResult) -> Void)?)` 
to make the `ApiBridge` defintion less verbose for all the REST Api methods?

Reply:
yes I should have used: RESTOperation.ResultListener? instead like you did for GraphQL